internal/cmd/auth.go

```diff
@@ -329,10 +332,15 @@ func authCallbackServer(state string) (authCallback, error) {
 	}, nil
 }
 
-func (a authCallback) Result() string {
-	result := <-a.ch
-	_ = a.server.Shutdown(context.Background())
-	return result
+func (a authCallback) Result() (string, error) {
+	select {
+	case result := <-a.ch:
+		_ = a.server.Shutdown(context.Background())
+		return result, nil
+	case <-time.After(10 * time.Minute):
```

Comment:
```suggestion
	case <-time.After(5 * time.Minute):
```

Sure sounds good to me